docs/security_guidelines.md

```diff
@@ -74,6 +74,19 @@ To mitigate XSS attacks:
 - [Trusted Types guide](https://web.dev/articles/trusted-types) (web.dev)
 - [Trusted Types API](https://developer.mozilla.org/en-US/docs/Web/API/Trusted_Types_API) (MDN)
 
+### Implement restrictions on framing
+
+Controlling whether your site can be embedded in another site using an [`<iframe>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) can help protect against [clickjacking](https://developer.mozilla.org/en-US/docs/Web/Security/Attacks/Clickjacking) and certain [cross-site leak attacks](https://xsleaks.dev/).
+
+If you don't need your site to be embeddable in other sites, or if you need only certain specific sites to embed your site, implement framing protection using the [`frame-ancestors`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/Content-Security-Policy/frame-ancestors) CSP directive and the [`X-Frame-Options`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/X-Frame-Options) HTTP response header.
```

Comment:
In most cases, you would not want a production site to be embeddable elsewhere, especially unknown origins.  I suggest the default recommendation would be to restrict, and provide guidance for if a user needs the site to be embeddable, and to set the configuration as narrow as possible to only exactly what they need, to minimize risk.

Reply:
-> https://github.com/w3c-cg/swag/pull/17/commits/7e4c3470d0662452a62dfa1e530c36ff38fc432b